stdlib/array.gr

```diff
@@ -386,6 +411,9 @@ provide let reduceRight = (fn, initial, array) => {
  * @param initial: The initial value to use for the accumulator on the first iteration
  * @param array: The array to iterate
  * @returns The final accumulator returned from `fn`
+ * 
+ * @example
+ * Array.reducei((a, b, i) => a + b + i, 100, [> 1, 2, 3, 4]) // 116
```

Comment:
It would again probably make sense to change this to use `==` given it doesnt return void, to keep things in line with the rest of the stdlib docs:
```suggestion
 * Array.reducei((a, b, i) => a + b + i, 100, [> 1, 2, 3, 4]) == 116
```

Reply:
Same reply as on .map.  I believe the == is not a proper use of == unless you in line // true.  Why not just show the value 116 like Oscar recommended.